Documentation/compilation.rst

```diff
@@ -5,15 +5,14 @@ In order to run SeisSol, you need to first install:
 
 -  Python (>= 3.5)
 -  Numpy (>= 1.12.0)
--  SCons (>= 3.0, for instructions see below)
 -  hdf5 (>= 1.8, for instructions see below)
 -  netcdf (C-Release) (>= 4.4, for instructions see below)
--  Intel compiler (>= 17.0, icc, icpc, ifort) or GCC (>= 5.0, gcc, g++, gfortran)
+-  Intel compiler (>= 18.0, icc, icpc, ifort) or GCC (>= 7.0, gcc, g++, gfortran)
 -  Some MPI implementation (e.g. OpenMPI)
 -  ParMETIS for partitioning
 -  libxsmm (libxsmm\_gemm\_generator) for small matrix multiplications
 -  PSpaMM (pspamm.py) for small sparse matrix multiplications (required only on Knights Landing or Skylake)
--  CMake (for compiling submodules ImpalaJIT and yaml-cpp)
+-  CMake (>3.15), for compiling submodules ImpalaJIT and yaml-cpp, and for SeisSol itself
```

Comment:
If cmake >3.15 is needed, this should be stated in the root level cmakefile -> otherwise it will fail with weird errors